native/android/src/main/java/com/nozbe/watermelondb/Database.kt

```diff
@@ -59,26 +55,40 @@ class Database(private val name: String?, private val context: Context) {
                 return it.getInt(it.getColumnIndex("count"))
             }
 
-    fun unsafeResetDatabase(): Boolean = context.deleteDatabase(name)
-
-    fun inTransaction(function: () -> Unit) = db.transaction { function() }
+//    fun unsafeResetDatabase() = context.deleteDatabase("$name.db")
```

Comment:
hm, what happened here? Why was this changed to manually deleting contents of the database?

Reply:
There are some problems with `context.deleteDatabase(name)` (mainly - it does not deletes database) until I figure it out I changed it in favour of manual deletions

Reply:
OK. Can you make a task for this? I think actually deleting the database should be more reliable and safer (the deleted data, potentially sensitive, is not garbage collected immediately)

Reply:
Sure thing!